web/src/typing_data.ts

```diff
@@ -3,63 +3,64 @@ import * as util from "./util";
 
 // See docs/subsystems/typing-indicators.md for details on typing indicators.
 
-const typist_dct = new Map();
-const inbound_timer_dict = new Map();
+const typist_dct: Map<string, number[]> = new Map();
+// eslint-disable-next-line no-undef
+const inbound_timer_dict: Map<string, NodeJS.Timeout | undefined> = new Map();
```

Comment:
`NodeJS.Timeout` certainly isn’t right in the browser. Try `ReturnType<typeof setTimeout>` instead.

`const m = new Map<A, B>()` is a bit cleaner than `const m: Map<A, B> = new Map()`.